download-subreddit-images.sh

```diff
@@ -24,10 +35,12 @@ while : ; do
         a=$(($a+1))
     done
     after=$(echo -n "$content"| jq -r '.data.after')
-    if [ -z $after ]; then
+    if [[ -z $after ]]; then
+        break
+    fi
+    if [[ $after == "null" ]]; then
```

Comment:
Thx for report, I fixed the problem in bf7ef7db8c6670edda827a7c96a6ce8c4e77dac9. You may remove this condition now.